src/box/lua/trigger.c

```diff
@@ -253,3 +253,146 @@ box_lua_trigger_init(struct lua_State *L)
 	luaL_register_type(L, event_trigger_iterator_typename,
 			   trigger_iterator_methods);
 }
+
+/** Old API compatibility. */
+
+/**
+ * Checks positional arguments for luaT_event_reset_trigger.
+ * Throws an error if the format is not suitable.
+ */
+static void
+luaT_event_reset_trigger_check_positional_input(struct lua_State *L, int bottom)
+{
+	assert(lua_checkstack(L, bottom + 2));
+	/* Push optional arguments. */
+	while (lua_gettop(L) < bottom + 2)
+		lua_pushnil(L);
```

Comment:
`lua_settop(L, bottom + 2)`

And `lua_checkstack` doesn't seem to be necessary.

Reply:
Use `lua_settop` now, removed `lua_checkstack`.